dev/ComboBox/ComboBox_themeresources.xaml

```diff
@@ -627,32 +663,47 @@
                             FlowDirection="{TemplateBinding FlowDirection}"
                             Foreground="{ThemeResource ComboBoxHeaderForeground}"
                             FontWeight="{ThemeResource ComboBoxHeaderThemeFontWeight}"
-                            Margin="{ThemeResource ComboBoxTopHeaderMargin}"
+                            Margin="{StaticResource ComboBoxTopHeaderMargin}"
                             TextWrapping="Wrap"
                             VerticalAlignment="Top"
                             Visibility="Collapsed"
                             x:DeferLoadStrategy="Lazy" />
+
+                        <Border x:Name="HighlightBackground"
+                            Grid.Row="1"
+                            Grid.Column="0"
+                            Grid.ColumnSpan="2"
+                            Margin="-4,-4"
+                            Background="{ThemeResource ComboBoxBackgroundFocused}"
+                            BorderBrush="{ThemeResource ComboBoxBackgroundBorderBrushFocused}"
+                            BorderThickness="{ThemeResource ComboBoxBackgroundBorderThicknessFocused}"
+                            contract7Present:CornerRadius="{ThemeResource ComboBoxHiglightBorderCornerRadius}"
```

Comment:
Should this be template bound to the setter so it is settable?

Reply:
Oh nevermind, it looks like `Background` is the settable one?

---
In reply to: [567047179](https://github.com/microsoft/microsoft-ui-xaml/pull/3938#discussion_r567047179) [](ancestors = 567047179)

Reply:
@karenbtlai ye, this one is for focus rect (it's custom for this control)